### PR TITLE
fix(cli-sunpath): remove analysis period options from sunpath wea subcommand

### PR DESCRIPTION
**Pull request by mostaphaRoudsari**
Since one can generate the wea file for part of the year that should be handled before using the honeybee-radiance command.

I realized the we have not really reviewed the CLI PRs. Starting from this one let's do the reviews and slowly develop a standard that we can enforce among all the projects.

### CODE REVIEW

File: honeybee_radiance/cli/sunpath.py [hunk 1]
```diff
@@ -27,44 +26,55 @@ def sunpath():
...
     ' positive values are to the East of Greenwich and negative values are to the West.'
     )
-@click.option('--north', default=0, type=float, show_default=True,
+@click.option(
+    '--north', default=0, type=float, show_default=True,
     help='Angle to north (0-360). 90 is west and 270 is east'
     )
-@click.option('--start-date', default='JAN-01', show_default=True,
+@click.option(
+    '--start-date', default='JAN-01', show_default=True,
```
Comment (AntoineDao) [inline honeybee_radiance/cli/sunpath.py:48]: Is it worth considering merging start-date and start-time into one input variable `start-date-time`? Dito for end-date.

File: honeybee_radiance/cli/sunpath.py [hunk 2]
```diff
@@ -156,32 +158,41 @@ def sunpath_from_wea(wea, north, folder, name, log_file, start_date, start_time,
...
 @sunpath.command('epw')
 @click.argument('epw', type=click.Path(exists=True))
-@click.option('--north', default=0, type=float, show_default=True,
+@click.option(
+    '--north', default=0, type=float, show_default=True,
     help='Angle to north (0-360). 90 is west and 270 is east'
     )
-@click.option('--start-date', default='JAN-01', show_default=True,
+@click.option(
+    '--start-date', default='JAN-01', show_default=True,
```
Comment (AntoineDao) [inline honeybee_radiance/cli/sunpath.py:166]: Dito previous comment on `start-date-time`

File: honeybee_radiance/cli/sunpath.py (hunk 1 above)
Comment (mostaphaRoudsari) [reply honeybee_radiance/cli/sunpath.py:48]: That can give a wrong impression to people. JAN-01-8:00 to JAN-15-20:00 will give an impression that every hour between Jan 1st at 8am to Jan 15 and 8pm will be included. However the time that will be included is the hours between 8am-8pm between Jan 1 to Jan 15. For instance Jan1 at 10 pm is not included.

**Comment by mostaphaRoudsari** [comment]
> I personally prefer using flags for both mandatory and optional arguments as it removes any issues with argument ordering if we have more than one argument.

That's exactly one of those cases that I was hoping to come up from these reviews. I do put the mandatory ones as arguments and the optional ones as options. Mainly because that was the suggestion by Click manual and this is how Radiance command line works which I have been very happy with.

@chriswmackey, how have you been dealing with this subject?

File: honeybee_radiance/cli/sunpath.py (hunk 1 above)
Comment (AntoineDao) [reply honeybee_radiance/cli/sunpath.py:48]: Right, that makes sense!

**Pull request merged**